.gitlab-ci/env.sh

```diff
@@ -2,46 +2,60 @@
 
 #
 
-export BRANCH_NAME=${BRANCH_NAME:=$CI_COMMIT_REF_NAME}
-export COMMIT=${COMMIT:=$CI_COMMIT_SHA}
+export BRANCH_NAME=${BRANCH_NAME:=$CI_COMMIT_REF_SLUG}
 export COMMIT_TAG=${COMMIT_TAG:=$CI_COMMIT_TAG}
+export COMMIT=${COMMIT:=$CI_COMMIT_SHA}
+export ENVIRONMENT=${ENVIRONMENT:="dev.factory"};
+export HASH_SIZE=${HASH_SIZE:=7}
 export JOB_ID=${JOB_ID:=$CI_JOB_ID}
-export HASH_SIZE=${HASH_SIZE}
-export ENVIRONMENT=${ENVIRONMENT:="dev"};
-export CLUSTER_NAME="dev.factory";
-BRANCH_NAME_HASHED=$(printf "${BRANCH_NAME}" | sha1sum | cut -c1-${HASH_SIZE} )
+export PROJECT_PATH=${PROJECT_PATH:=$CI_PROJECT_PATH}
```

Comment:
I don't see where it is used

Reply:
Should be use in the standalone version of the github deploy keys scripts